Accommodate whitespace in header value

lisac-usds opened the pull request:
## Summary of changes

A few changes that made sense to me (but may be incorrect):
- accommodate whitespace in the header value (eg accept `includeSubDomains ;`)  (why: it's not unheard of to have extra whitespace - 2 examples: domains whitehouse.gov and dhs.gov )
- change a `=` to a `&=` (logic bug?) 
- reduce files included in the archive 
- edit sample usage code snippet

lisac-usds committed:
exclude test and util files from the lambda archive

lisac-usds committed:
update sample usage - specify the script name

why: I'm not able to get the expected output unless I include the script name

lisac-usds committed:
update to consider previous value

lisac-usds committed:
allow extra whitespace in the sts header tokens

lisac-usds commented:
Also, I had some questions about hsts headers, based on some of the domains I saw getting flagged:

1. is the `preload` a mandatory part of a proper hsts header?  
For instance, domains dhs.gov, fema.gov, and usajobs.gov do not include it.
2. is a max-age of 1 year a mandatory part of a proper hsts header? (would 180 days suffice?)
For instance, domain reddit.com returns `max-age=15552000; includeSubDomains; preload`

mark-from-usds commented on package_lambda:
```diff
@@ -5,6 +5,6 @@ OUTPUT_ABSOLUTE=${PWD}/https_checker_lambda_${TIMESTAMP}.zip
 VENV=$(pipenv --venv)
 pushd ${VENV}/lib/python3.7/site-packages
-zip -r9 ${OUTPUT_ABSOLUTE} ./*
+zip -r9 ${OUTPUT_ABSOLUTE} ./* -x "./test/*" "./package_lambda"
```
At this point $PWD is site-packages so if there's something named test or package_lambda there, it's not from our repo, and it might be related to a dependency, so I'd rather not exclude it.

lisac-usds replied:
Ah, now I understand. Thanks!
Un-did the change in 2c66182.

mark-from-usds reviewed (changes requested):
Except for one nitpick at the bottom, this is all great!

lisac-usds committed:
do not exclude test&util from the lambda archive (revert 6856c67)

why: At this point $PWD is site-packages so if there's something named test or package_lambda there, it's not from our repo, and it might be related to a dependency; safer not to exclude it
